Fix mistyped pattern name in css syntax

phynalle opened the pull request:
### Description of the Change
Some parameters inside pseudo class **dir** haven't been colorized. It's trivial but it should be fixed.
The reason is the mistypo of pattern name and I just fixed it.
<!--

We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.

-->

### Alternate Designs

<!-- Explain what other alternates were considered and why the proposed version was selected -->
None

### Benefits

<!-- What benefits will be realized by the code change? -->
We can see pattern colorizing inside dir pseudo class as intended.

### Possible Drawbacks

<!-- What are the possible side-effects or negative impacts of the code change? -->
None

### Applicable Issues

<!-- Enter any applicable Issues here -->
None

50Wliu commented:
Thanks!  Could you add some specs to this as well?

phynalle commented:
What kind of spec do you need? I think the description is enough because I just corrected a mistypo.

50Wliu commented:
I mean specs as in tests: http://flight-manual.atom.io/hacking-atom/sections/writing-specs/
You add them to [this file](https://github.com/atom/language-css/blob/master/spec/css-spec.coffee).

phynalle commented:
I think specs should be remained.
I fixed ignoring some patterns which colorize parameter of :dir().
In current spec file, I think the specs about it are all written. So no additional spec is needed.
That is also the reason why the misspelling haven't been discovered.
Well, I can put any specs related to patterns I corrected but they wouldn't be allowed in css grammar.
